docs/source/example_notebooks/gcm_counterfactual_medical_dry_eyes.ipynb

```diff
@@ -0,0 +1,291 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Counterfactual analysis in a medical case"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we examine a case where we want to ask counterfactual questions, for events that have already taken place. We focus on a tele-medicine example related to eye-sight problems, where we know the causal structure of three observed variables and we want to ask counterfactual questions based of the type \"What would have happened had I followed a different approach of what the tele-medicine app had proposed?\".\n",
+    "\n",
+    "More specifically, we consider the following case.\n",
+    "Alice who experiences intense eye-dryness decides to use a tele-medicine online platform, as she is not able to visit an eye doctor where she lives. She goes through the steps of reporting her medical history (whether she has a rare allergy, for Alice Condition = 1) and the platform in the end recommends to her two possible eye drops, with slightly different ingredients (option 'A' and option 'B'). Alice does a quick search online and she finds that the are many positive reviews for option 'A'. Nevertheless, she decides to use option 'B' (Treatment = 2) as her mother has also used it in the past and it had positive results. After a couple of days Alice's vision becomes much better and her symptoms start to disappear. However she is very curious what would have happened if she had used the very popular option 'A' instead, or even did nothing.\n",
```

Comment:
> (whether she has a rare allergy, for Alice Condition = 1)
Does this mean that the variable "Condition" is 1 if one has a rare allergy and 0 otherwise?
It might be more clear if you introduced the names of the variables (e.g. Treatment and Condition) before the outcomes they can take (e.g. option A and option B for Treatment)

Reply:
Good point, I slightly introduced the introduction. We probably don't need to mention variables names at all at that point.